### ISSUE TITLE
Problem with validation loss value for BC (GMM)

### ISSUE THREAD

**Issue by Ariostgx** [ARISE-Initiative/robomimic#12]
Hi,

Thanks for the great work. I tried to run the BC algorithm with a GMM base and found in the Tensorboard record that the "validation loss" is significantly larger than the "training loss" (1e6 vs ~10).

This confused me to think at first there is a huge gap between rollout on the training and validation set. However, later I noticed that you will set the "low_noise_eval" to be True during evaluation, which makes all the scales be 1e-4, which makes the validation loss much higher than the training loss.

In my opinion, the validation loss should be computed with a predicted scale, otherwise, it will mislead the user by contrasting it with the training loss. I wonder if this is an intentional choice? Thanks!

**Comment by amandlek** [comment]
This is a great point! Indeed, when we computed the validation losses and used them for policy selection in our paper, we computed the loss when using the predicted GMM mode scales, but the default behavior in our repository is to use the fixed scales during evaluation (as opposed to training), and validation is considered to be evaluation (a standard practice in PyTorch model evaluation is to turn off gradients and put the model into "eval" mode, so we wanted to be consistent). 

You could probably customize the behavior of the GMM policy network to make the distinction between validation and rollouts for your purposes.
